appveyor.yml

```diff
@@ -46,9 +46,9 @@ environment:
         SHARED: OFF
         DISABLED_TESTS: ""
         COMPILER_PATH: ""
-      - APPVEYOR_BUILD_WORKER_IMAGE: "Visual Studio 2017"
+      - APPVEYOR_BUILD_WORKER_IMAGE: "Visual Studio 2019"
         BUILD_SYSTEM: CMake
-        PRJ_GEN: "Visual Studio 15 2017 Win64"
+        PRJ_GEN: "Visual Studio 15 2019 Win64"
```

Comment:
The deprecated architecture selection via the generator name has been removed from the VS 2019 generator, it has to be specified with "-A x64" now: https://cmake.org/cmake/help/latest/generator/Visual%20Studio%2016%202019.html

Reply:
Aha, thanks. I'll amend!